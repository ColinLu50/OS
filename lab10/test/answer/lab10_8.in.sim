5780
525
552
354
552
353
