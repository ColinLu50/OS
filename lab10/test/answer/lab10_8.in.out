FCFS
195 -> 9 -> 152 -> 327 -> 166 -> 0 -> 22 -> 1 -> 346 -> 179 -> 265 -> 311 -> 283 -> 256 -> 234 -> 2 -> 85 -> 357 -> 212 -> 36 -> 148 -> 24 -> 311 -> 232 -> 233 -> 192 -> 227 -> 28 -> 199 -> 117 -> 222 -> 131 -> 290 -> 257 -> 266 -> 357 -> 249 -> 63 -> 54 -> 141 -> 0 -> 192 -> 304 -> 245 -> 295 -> 94 -> 344 -> 81
5780
SSTF
195 -> 192 -> 192 -> 199 -> 212 -> 222 -> 227 -> 232 -> 233 -> 234 -> 245 -> 249 -> 256 -> 257 -> 265 -> 266 -> 283 -> 290 -> 295 -> 304 -> 311 -> 311 -> 327 -> 344 -> 346 -> 357 -> 357 -> 179 -> 166 -> 152 -> 148 -> 141 -> 131 -> 117 -> 94 -> 85 -> 81 -> 63 -> 54 -> 36 -> 28 -> 24 -> 22 -> 9 -> 2 -> 1 -> 0 -> 0
525
SCAN
195 -> 192 -> 179 -> 166 -> 152 -> 148 -> 141 -> 131 -> 117 -> 94 -> 85 -> 81 -> 63 -> 54 -> 36 -> 28 -> 24 -> 22 -> 9 -> 2 -> 1 -> 0 -> 0 -> 199 -> 212 -> 222 -> 227 -> 232 -> 233 -> 234 -> 245 -> 249 -> 256 -> 257 -> 265 -> 266 -> 283 -> 290 -> 295 -> 304 -> 311 -> 327 -> 344 -> 346 -> 357
552
C-SCAN
195 -> 192 -> 179 -> 166 -> 152 -> 148 -> 141 -> 131 -> 117 -> 94 -> 85 -> 81 -> 63 -> 54 -> 36 -> 28 -> 24 -> 22 -> 9 -> 2 -> 1 -> 0 -> 0 -> 358 -> 357 -> 346 -> 344 -> 327 -> 311 -> 304 -> 295 -> 290 -> 283 -> 266 -> 265 -> 257 -> 256 -> 249 -> 245 -> 234 -> 233 -> 232 -> 227 -> 222 -> 212 -> 199
354
LOOK
195 -> 192 -> 179 -> 166 -> 152 -> 148 -> 141 -> 131 -> 117 -> 94 -> 85 -> 81 -> 63 -> 54 -> 36 -> 28 -> 24 -> 22 -> 9 -> 2 -> 1 -> 0 -> 199 -> 212 -> 222 -> 227 -> 232 -> 233 -> 234 -> 245 -> 249 -> 256 -> 257 -> 265 -> 266 -> 283 -> 290 -> 295 -> 304 -> 311 -> 327 -> 344 -> 346 -> 357
552
C-LOOK
195 -> 192 -> 179 -> 166 -> 152 -> 148 -> 141 -> 131 -> 117 -> 94 -> 85 -> 81 -> 63 -> 54 -> 36 -> 28 -> 24 -> 22 -> 9 -> 2 -> 1 -> 0 -> 357 -> 346 -> 344 -> 327 -> 311 -> 304 -> 295 -> 290 -> 283 -> 266 -> 265 -> 257 -> 256 -> 249 -> 245 -> 234 -> 233 -> 232 -> 227 -> 222 -> 212 -> 199
353
